helmfile.d/charts/prometheus-alerts/templates/alerts/falco-alerts.yaml

```diff
@@ -21,7 +21,7 @@ spec:
       annotations:
         message: 'Pod: {{`{{ $labels.pod }}`}}, Rule: {{`{{ $labels.rule }}`}}'
       expr: |-
-        falco_events != 0
+        falcosecurity_falcosidekick_falco_events_total != 0
```

Comment:
Are these equivalent when it comes to labels as well, or perhaps fairly close? I want to ensure that the alert information is not significantly worse (if it is better, then that is of course good :slightly_smiling_face: )

Also, we currently do not enable falco sidekick by default in sc ([config](https://github.com/elastisys/compliantkubernetes-apps/blob/main/config/sc-config.yaml#L123), [values](https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile.d/values/falco/falco-common.yaml.gotmpl#L380C29-L380C35)). So if we want this then we would need sidekick to be enabled by default in sc as well.

Reply:
They're fairly similar both when it comes to information and labels. 

Thanks for the catch, I've now enabled falco sidekick by default.

Reply:
> They're fairly similar both when it comes to information and labels.

Ok, thanks :+1: 

> Thanks for the catch, I've now enabled falco sidekick by default.

I see that you enabled it by changing `falco.alerts.enabled=true` in sc as well. But I think that this will also mean that sidekick will start to send alerts about falco events in sc to alertmanager. So we would start to get falco alerts for sc from both sidekick and from prometheus alerts. I don't think that's really what we want. Could you have a separate config option to enable sidekick and one to have sidekick send alerts to alertmanager? Then we can enable just the first in sc and both in wc. Do you see some other solution?

Reply:
I have added type none as a valid type and tried running falco sidekick without any type but managed to see prometheus still picking metrics as usual from falco sidekick when falco event generator was installed in sc.

Reply:
Thanks for the updates!